README.md

```diff
@@ -7,44 +7,91 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)l)
+  - [Usage](#usage)
 - [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements))
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Mobile version project
-In this project i designed different cards
-of a mobile version , check my github to discover more .
-Thank you 
+# 📖 [Microverse Desktop version project] 
+
+The project of Desktop version is related on how content
+are displayed on desktop and mobile view and both are responsive
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-  <summary>HTML/CSS</summary>
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
 </details>
 
 <!-- Features -->
 
-- **[added html]**
-- **[added css]**
+- **[HTML]**
+- **[CSS]**
+
+<!-- LIVE DEMO -->
+
+-(https://github.com/Dorcas126/Microverse-Portofolio-project/tree/desktop)
+
+<!-- GETTING STARTED -->
+
+To get a local copy up and running, follow these steps.
+
+Just go to my github account using the above ,you will access the project.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Any code editor like 
+Vs code
+Sublime
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/Dorcas126/Microverse-Portofolio-project/tree/desktop
 
 ### Usage
 
 To run the project, execute the following command:
 
-1.Linters
-2.HTML/CSS
+HTML
+CSS
 
 <!-- AUTHORS -->
 
-## 👥 Authors <a name="authors"></a>
-
 👤 **Author1**
 
 - GitHub: [@Dorcas126]
-- Twitter: [@DorcasDorc2]
+- Twitter: [@DorcasDorca]
 - LinkedIn: [Uwimbabazi Dorcas]
```

Comment:
- you did great by adding your user name but make sure you make them a link so that they can direct you to you profiles
:+1: